bokeh/models/glyphs.py

```diff
@@ -15,26 +15,16 @@
 
 @abstract
 class Glyph(PlotObject):
-    """ Base class for all glyphs/marks/geoms/whatever-you-call-'em in Bokeh.
-
-    """
+    """ Base class for all glyph models. """
```

Comment:
Previously you were pushing for the former style. What changed? Anyway, I'm more than +1 on on wasting vertical space.